packages/ember-runtime/lib/mixins/mutable_array.js

```diff
@@ -388,7 +389,9 @@ export default Mixin.create(EmberArray, MutableEnumerable, {
     @public
   */
   addObject(obj) {
-    if (!this.contains(obj)) {
+    var included = isEnabled('ember-runtime-enumerable-includes') ? this.includes(obj) : this.contains(obj);
```

Comment:
Impacts on `addObject` are not so small because using `includes` instead of  `contains` here will allow to add `null` and `undefined` values into an Enumerable ... Is it expected ?